packages/thirdweb/src/chainsaw/endpoints/getNFTsByCollection.ts

```diff
@@ -0,0 +1,66 @@
+import type { ThirdwebClient } from "../../client/client.js";
+import type { Address } from "../../utils/address.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import { formatChainsawNFTs } from "../formatter.js";
+import { addPagingToRequest } from "../paging.js";
+import type {
+  ChainsawPagingParams,
+  ChainsawResponse,
+  NFTs,
+  NFTsData,
+} from "../types.d.ts";
+import { getNftsByCollectionEndpoint } from "../urls.js";
+
+export type GetNFTsGroupBy = "ownerAddress";
+
+export type GetNFTsByCollectionParams = {
```

Comment:
Prettify type

Reply:
Done in [3a5bdf8](https://github.com/thirdweb-dev/js/pull/3741/commits/3a5bdf8569c9f88a2f276508dc6a6b6a3068cf8d)